nbs/18_callback.fp16.ipynb

```diff
@@ -202,18 +202,49 @@
     "    def after_loss(self): self.autocast.__exit__(None, None, None)\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        flat_master:bool=False, # Flatten fp32 params into a vector for better performance
</pre><p>Better performance and half memory? or that last is not always true?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/muellerzr/fastai-docment-sprint/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
No, it is a lot more nuanced. This would flatten the fp32 wights into a single vector so we can update them all at once as opposed to in a loop.